src/app/hooks/utils/CompanyDTO.ts

```diff
@@ -0,0 +1,8 @@
+type Company = {
```

Comment:
As mentioned with the naming conventions:
```suggestion
interface Company = {
```

Reply:
In that case, it should be interface Company { (whithout the = sign).
Committed the changes